packages/apollo-server-express/src/ApolloServer.ts

```diff
@@ -67,7 +70,21 @@ const fileUploadMiddleware = (
   }
 };
 
+interface ExpressContext {
+  req: express.Request;
+  res: express.Response;
+}
+
+export interface ApolloServerExpressConfig extends Config {
+  cors?: CorsOptions | boolean;
```

Comment:
@cheapsteak I think bringing this `cors` property into the `ApolloServerExpress` config may actually go against the existing patterns for users of `apollo-server-express`, which I believe has typically asked for `cors` to be set as part of `applyMiddleware`.

For example, as https://github.com/apollographql/apollo-server/issues/1882 points out, there's a typing issue on the `ApolloServer` exported from the standalone `apollo-server` package (which does _not_ require or have  `applyMiddleware` since it's "standalone") that seems to omit the `cors` property.

While the integrations pass their `cors` settings via `applyMiddleware`, the standalone doesn't have `applyMiddleware`, so it was necessary to provide desired `cors` settings via the constructor.

Currently, this new typing seems to almost encourage `apollo-server-express` users to set their `cors` in the constructor as well, even though all the other integrations require `cors` to be set in the `applyMiddleware`.  In fact, I don't believe that setting `cors` via `apollo-server-express`'s `ApolloServer` constructor would actually work since it's only in the `apollo-server` implementation that we set `this.cors = config && config.cors;`.

Reply:
Great point, appreciate the thorough breakdown
Opened a PR to address this - https://github.com/apollographql/apollo-server/pull/2373